hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientNearCacheTestSupport.java

```diff
@@ -629,6 +631,46 @@ public void run() {
         });
     }
 
+    protected void testNearCacheMemoryCostCalculation(InMemoryFormat inMemoryFormat, int threadCount) {
```

Comment:
That a good point, but I think we should check for it. I've added an explicit check that there are no memory costs being calculated for `OBJECT` in-memory format.